doc/internal-marble-tests.md

```diff
@@ -0,0 +1,188 @@
+> NOTE: This document refers to the writing marble tests for the **RxJS repo internals** and is intended for anyone wishing to help maintain the RxJS repo. **Users of RxJS** should instead view the [guide for writing marbles tests for applications](./marble-testing.md). The major difference is that the behavior of the TestScheduler differs between manual usage and using the `testScheduler.run(callback)` helper.
+
+# Writing Marble Tests
```

Comment:
This file is just a copy and paste from the existing old [writing-marble-tests.md](https://github.com/ReactiveX/rxjs/blob/76b7e272433a5c94832b2a2f945af526236e0fde/doc/writing-marble-tests.md), with the added disclaimer at the top.